### PR TITLE
Upgraded packages to use .NET 5.0

### PR DESCRIPTION
**Pull request by niemyjski**
This is a breaking change for the WinForms and WPF nuget packages as it does drop .net core 3.1 in favor of .NET 5

### CODE REVIEW

File: global.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "sdk": {
-    "version": "3.1.100",
+    "version": "5.0.100",
     "rollForward": "latestMinor"
```
Comment (niemyjski) [inline global.json:4]: @ejsmith we were doing `latestMajor` in exceptionless.

**Comment by benaadams** [comment]
Bump `net452` and `v4.5.2` to `net462` and `v4.6.2` also?

**Comment by benaadams** [comment]
> Bump `net452` and `v4.5.2` to `net462` and `v4.6.2` also?

Made PR for that https://github.com/exceptionless/Exceptionless.Net/pull/250

File: global.json (hunk 1 above)
Comment (ejsmith) [reply global.json:4]: Yeah, latestMinor is probably the correct choice. That's what we switched to in other projects.

**Comment by niemyjski** [comment]
@benaadams Thanks!

**Pull request merged**